medallion/filters/basic_filter.py

```diff
@@ -45,6 +44,16 @@ class BasicFilter(object):
 
     def __init__(self, filter_args):
         self.filter_args = filter_args
+        self.match_type = self.filter_args.get("match[type]")
+        if self.match_type:
+            self.type_ = self.match_type.split(",")
```

Comment:
Whoops, `"".split(",")` returns `['']` instead of `[]`

I still think it would make the code easier to read if we reused the variable though (also in the other cases I mentioned in the previous comment):

```suggestion
        self.match_type = self.filter_args.get("match[type]")
        if self.match_type:
            self.match_type = self.match_type.split(",")
```